File: 04-05/func_for_n-m.py
def funcFor(n, m):
    if n >= m:
        print(n)
    else:
        print(n)
        funcFor(n+1, m)

if __name__ == "__main__":
    print("Digite dois números (o segundo maior que o primeiro)")
    funcFor(int(input("Primeiro: ")), int(input("Segundo: ")))
    

File: 04-05/funcFor_n-m.hs
funcFor n m
    | n >= m = print(n)
    | otherwise = do 
        print(n)
        funcFor (n+1) m


main :: IO ()
main = do 
    putStrLn "Digite um número:"
    n <- readLn
    putStrLn "Digite um número maior:"
    m <- readLn
    putStrLn "Contagem:"
    funcFor n m